lib/services/v2.0/nomination_pools/txs/update_roles.dart

```diff
@@ -5,8 +5,6 @@ class KC2UpdateRolesTxV1 extends KC2Tx {
   PoolId poolId;
   MapEntry<ConfigOption, String?> root;
   MapEntry<ConfigOption, String?> nominator;
-
-  /// @HolyGrease - please add code doc. What is a bouncer?
```

Comment:
Bouncer can change the pools state and kick members if the pool is blocked. For more info about all pool roles please search for `PoolRoles`